src/DataLoader/DataLoader.csproj

```diff
@@ -0,0 +1,7 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;netstandard1.3;net45</TargetFrameworks>
+  </PropertyGroup>
+
+</Project>
```

Comment:
Ok, I didn't realize we were talking about putting it all in to the main project. But I'm fine with that. I've always been a bit skeptical of the use of a DataLoader outside of GraphQL anyway. And it could always be separated out later.

Reply:
If I put this in the main project, I'll need to target netstandard1.3 instead of netstandard1.1. Is that a problem?

Reply:
Its probably time to update to it. 🎉